packages/block-library/src/avatar/edit.js

```diff
@@ -222,7 +222,8 @@ const UserEdit = ( { attributes, context, setAttributes, isSelected } ) => {
 };
 
 export default function Edit( props ) {
-	if ( props?.context?.commentId ) {
+	// We set commentId as null if we are on the Site Editor. See "../src/comment-template/edit.js L215"
```

Comment:
Line numbers change all the time, so let's remove it.

```suggestion
	// We set commentId as null if we are on the Site Editor. See ../src/comment-template/edit.js
```

(Or maybe even remove the file reference.)

Maybe a bit more context would help, since it's not totally clear that the comment only refers to the second part of the condition (after the `||`) 🤔 

Perhaps something like

```suggestion
	// Don't show the Comment Edit controls if we have a comment ID set, or if we're in the Site Editor (where it is `null`).
```